packages/sdks/src/components/content-variants/content-variants.lite.tsx

```diff
@@ -82,15 +82,13 @@ export default function ContentVariants(props: VariantsProviderProps) {
     },
   });
 
-  onUpdate(() => {
+  onInit(() => {
     useTarget({
       angular: () => {
-        if (props.content) {
-          state.shouldRenderVariants = checkShouldRenderVariants({
-            canTrack: props.canTrack,
-            content: props.content,
-          });
-        }
+        state.shouldRenderVariants = checkShouldRenderVariants({
+          canTrack: getDefaultCanTrack(props.canTrack),
+          content: props.content,
+        });
       },
```

Comment:
2 things:

- this is an SDK change, so we'd need a `changesets` entry in this PR
- why do we need this `onInit` at all? The initial value is already set to:

```ts
shouldRenderVariants: checkShouldRenderVariants({
      canTrack: getDefaultCanTrack(props.canTrack),
      content: props.content,
    }),
```

Given the recent improvements in Mitosis, wouldn't this initial value be initialized properly in Angular?

Reply:
added the changeset

right, we should handle the state initialization correctly inside mitosis. should i make a follow-up pr where I will clean up this after we've added it to mitosis?